hubblestack/extmods/modules/win_pulsar_winaudit.py

```diff
@@ -0,0 +1,554 @@
+#win_notify
+'''
+This will setup your computer to enable auditing for specified folders inputted into a yaml file. It will
+then scan the event log for changes to those folders and report when it finds one.
+'''
+
+
+from __future__ import absolute_import
+
+import collections
+import datetime
+import fnmatch
+import logging
+import os
+import glob
+import yaml
+import re
+
+import salt.ext.six
+import salt.loader
+
+log = logging.getLogger(__name__)
+DEFAULT_MASK = ['ExecuteFile', 'Write', 'Delete', 'DeleteSubdirectoriesAndFiles', 'ChangePermissions',
+                'TakeOwnership'] #ExecuteFile Is really chatty
+DEFAULT_TYPE = 'all'
+
+__virtualname__ = 'pulsar'
```

Comment:
Because we're keeping this old module around, it's going to try to load both the old and the new modules under the same `__virtualname__`. We should change one of them to avoid this conflict.